network/tests/topology.rs

```diff
@@ -115,15 +102,14 @@ async fn spawn_nodes_in_a_mesh() {
         ..Default::default()
     };
     let mut nodes = test_nodes(N, setup).await;
-    connect_nodes(&mut nodes, Topology::Mesh);
-    start_nodes(&nodes).await;
+    connect_nodes(&mut nodes, Topology::Mesh).await;
 
     // Set the sleep interval to 200ms to avoid locking issues.
     // Density measurement here is proportional to the min peers: if every node in the network
     // only connected to the min node count, the total number of connections would be roughly 10
-    // percent of the total possible. With 50 nodes and min at 5 connections each this works out to
-    // be 125/1225 ≈ 0.1.
-    wait_until!(15, network_density(&nodes) >= 0.1, 200);
+    // percent of the total possible. With 25 nodes and min at 5 connections each this works out to
```

Comment:
this change is done so that the CI has an easier time with the test, right?

Reply:
It's more accurate (i.e. restoring the original intent of the test), the peer count was lowered without lowering the MIN_PEERS which changed the density expectation. Forgot to add the expected density calculation to these tests.